examples/e14_slash_commands/src/main.rs

```diff
@@ -14,6 +14,8 @@ struct Handler;
 impl EventHandler for Handler {
     async fn interaction_create(&self, ctx: Context, interaction: Interaction) {
         if let Interaction::ApplicationCommand(command) = interaction {
+            println!("Received command interaction: {:#?}", command);
```

Comment:
Is this supposed to be left here?

Reply:
Yes, it's useful to see if interactions are deserialized properly; specifically, the {name, description}_{localized, localizations} in this PR. Similar reason to why registered commands are printed too. Also, it's useful to see received parameters and how they're structured, since this example doesn't handle most received interactions in code